Makefile.am

```diff
@@ -22,7 +22,6 @@ CODE_COVERAGE_OUTPUT_FILE = libseccomp.lcov.info
 CODE_COVERAGE_OUTPUT_DIRECTORY = libseccomp.lcov.html.d
 CODE_COVERAGE_IGNORE_PATTERN = \
 	*/usr/include/* \
-	*/src/arch-syscall-check.c \
```

Comment:
Is this hunk relevant to this PR?

Reply:
@kolyshkin indeed it is see in https://github.com/t0b3/libseccomp/actions/runs/12359993960/job/34493901917#step:5:9488 coverage fails otherwise